docs/download_files.md

```diff
@@ -2,6 +2,9 @@
 
 The ScPCA Portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
 These files are delivered as a zip file.
+When you uncompress the zip file, the root directory name of your download will include the date you accessed the data on the ScPCA Portal.
+We recommend you record this date in case there are future updates to the Portal that change the underlying data or if you need to cite the data in the future (see {ref}`How to Cite <citation:how to cite>` for more information).
+<!-- TODO: Add link to CHANGELOG -->
```

Comment:
I think we need to do this before release. Copying text from `FAQ`?
```suggestion
Please see our {ref}`CHANGELOG <CHANGELOG:CHANGELOG>` for a summary of changes that impact downloads from the Portal.
```

Reply:
ah good catch, im going to commit this to test

Reply:
yup this looks good